ee/clickhouse/sql/trends/breakdown.py

```diff
@@ -7,7 +7,7 @@
                 (
                     SELECT breakdown_value
                     FROM (
-                        SELECT %(values)s as breakdown_value
+                        SELECT CAST(%(values)s, 'Array(string)') as breakdown_value
```

Comment:
Would be good to have a regression test if possible

Reply:
We already had one - it just didn't raise an error. The first commit made it fail loudly.